server/i18n/en.json

```diff
@@ -8878,11 +8878,11 @@
   },
   {
     "id": "model.config.is_valid.elastic_search.enable_autocomplete.app_error",
-    "translation": "Search EnableIndexing setting must be set to true when Elasticsearch EnableAutocomplete is set to true"
+    "translation": "Search EnableIndexing setting must be set to true when EnableAutocomplete is set to true"
```

Comment:
I think the goal here is to retain the setting name as is since it's not changing (e.g. `ElasticsearchSettings.EnableIndexing` but inject it in the string through a variable `{{.SettingName}}` so that it doesn't get translated.